multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -281,7 +281,7 @@ def _fit_x(self, X: pd.DataFrame):
                     generator = self._feature_generators[col_name]
                     generator.fit(np.expand_dims(processed_data.to_numpy(), axis=-1))
                     self._numerical_feature_names.append(col_name)
-            elif col_type.startswith(IMAGE):
+            elif col_type.startswith(IMAGE) or col_type == ROIS:
```

Comment:
Maybe mark a TODO that makes `ROIS` an independent modality and uses the `transform_multimodal()` in which we can call `transform_image()` and `transform_rois()`.